src/__tests__/index.js

```diff
@@ -195,7 +195,7 @@ import Demo from './demo.blah'
 
   assert.match(
     error.message,
-    `error: [plugin: inMemory] Invalid loader: "blah" (valid: js, jsx, ts, tsx, css, json, text, base64, dataurl, file, binary)`,
+    `ERROR: [plugin: inMemory] Invalid loader value: "blah"`,
```

Comment:
I figuered since esbuild might add other loaders by default so we don't want this test to fail when that happens.